api/lib/domain/models/Organization.js

```diff
@@ -64,6 +64,10 @@ class Organization {
     return Boolean(this.tags.find((tag) => this.isSco && tag.name === Tag.AGRICULTURE));
   }
 
+  get isCFA() {
+    return Boolean(this.tags.find((tag) => this.isSco && tag.name === Tag.CFA));
+  }
```

Comment:
```suggestion
  get isAgricultureCFA() {
    return Boolean(this.tags.find((tag) => this.isSco && this.isAgriculture && tag.name === Tag.CFA));
  }
```
+ ajout de test associé :)

Reply:
ah tu as laissé la logique agri + CFA côté front. En fait j'hésite, je sais pas ce qui est le mieux

Reply:
c'est une logique purement affichage en vrai je pense, c'est ok de laisser ça dans le front